src/onefetch/cli.rs

```diff
@@ -255,6 +277,25 @@ impl Cli {
             None
         };
 
+        if let Some(should_hide_logo) = matches.value_of("hide-logo").or(Some("auto")) {
+            if should_hide_logo == "true" {
+                art_off = true;
+            } else if should_hide_logo == "false" {
+                art_off = false;
+            } else if !art_off {
+                if let Some((width, _)) = term_size::dimensions_stdout() {
+                    art_off = width <= max_term_size;
+                } else {
+                    std::println!(
```

Comment:
```suggestion
                    std::eprintln!(
```
Prints to STDERR instead of STDOUT. Might be prefer if a user wants to do something like `onefetch 2> error.log` to hide error messages in the terminal.
Also, see question above about using `std::`.

Reply:
Alright